lib/liquid_voting/voting_results.ex

```diff
@@ -67,6 +67,27 @@ defmodule LiquidVoting.VotingResults do
     )
   end
 
+  @doc """
+  Publishes voting result changes to Absinthe's pubsub for all results related
+  to a specific user's votes.
+
+  This is called by both the create_delegation/3 and delete_delegation/3 defs in
+  the absinthe layer lib/liquid_voting/reolvers/delegations.ex file. This ensures
+  that related voting results AND Absinthe's pubsub are updated accordingly.
+
+  ## Example
+
+      iex> publish_voting_result_changes_for_participant("377ead47-05f1-46b5-a676-f13b619623a7", "a6158b19-6bf6-4457-9d13-ef8b141611b4")
+      :ok
+
+  """
+  def publish_voting_result_changes_for_participant(participant_id, organization_id) do
```

Comment:
I chose to abstract out this def (which is called by both the delegations.ex resolver create_delegation & delete_delegation functions) and place it here, in LiquidVoting.VotingResults, as this seemed most logical to me. It could also live in the resolver file, however.  
Thoughts on this choice?

Reply:
This falls in that gray area of concerns that Niklas brought up a while back: it is a `VotingResults` concern, semantically, but it's touching the absinthe layer from the business logic / data layer. And granted, we were already doing that with `VotingResults.publish_voting_result_change` beforehand.

We can go with this for now, but we do need to refactor soonish. This mixing of the layers is confusing

Reply:
> We can go with this for now, but we do need to refactor soonish. This mixing of the layers is confusing

Understood. We could look at this when we pair on #153 :+1: